domain/credential/state/types.go

```diff
@@ -105,10 +104,23 @@ type CredentialAttribute struct {
 	Value string `db:"value"`
 }
 
+// dbCloudName represents the name of a cloud.
+type dbCloudName struct {
```

Comment:
We've got a mix of with/without the "db" prefix in here.

Reply:
As per the above comment, this resolves a naming issue